### PR TITLE
[ADD] Cron pour réparer livré incorrect sur bon de commande et feuilles de temps 

### PR DESCRIPTION
**Pull request by robinkeunen**
https://gestion.coopiteasy.be/web#id=1040&view_type=form&model=project.task&action=479

needs: https://github.com/OCA/OCB/pull/851

### CODE REVIEW

**Comment by robinkeunen** [comment]
@PierrickBrun

File: coopiteasy_custom/models/sale_timesheet.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+from openerp import api, fields, models, _
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    @api.model
+    def reset_so_line(self):
+        lines = self.search([])
```
Comment (houssine78) [inline coopiteasy_custom/models/sale_timesheet.py]: maybe just take into account line from open SO or uninvoiced lines would be better than recomputing analytic lines for invoiced lines ?

Comment (robinkeunen) [reply coopiteasy_custom/models/sale_timesheet.py]: done, check this out.

File: coopiteasy_custom/data/cron.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright 2017-2018 Coop IT Easy
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
```
Comment (houssine78) [inline coopiteasy_custom/data/cron.xml:5]: do we put copyright in xml file ?

File: coopiteasy_custom/data/cron.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+<odoo>
+    <data noupdate="0">  <!-- fixme-->
+        <record id="cie_fix_account_analytic_line_so_line" model="ir.cron">
+            <field name="name">cie fix account_analytic_line so_line</field>
+            <field name="active" eval="False"/>
+<!--            <field name="user_id" ref="base.user_root"/>-->
```
Comment (houssine78) [inline coopiteasy_custom/data/cron.xml]: if you don't plan to define a value for it. remove this line

**Review by houssine78 — COMMENTED** [review]
small remarks. can be merged

File: coopiteasy_custom/data/cron.xml (hunk 2 above)
Comment (robinkeunen) [reply coopiteasy_custom/data/cron.xml:5]: Yup but should be 2019

File: coopiteasy_custom/data/cron.xml (hunk 3 above)
Comment (robinkeunen) [reply coopiteasy_custom/data/cron.xml]: kay babe

**Pull request merged**